nginx.conf

```diff
@@ -34,6 +34,12 @@ server {
         rewrite ^/(.*[^/])$ /$1/ permanent;
     }
 
+    # Don't cache status check
+    location ~ ^/_status {
+      try_files $uri $uri.html $uri/ =404;
+      expires epoch;
```

Comment:
Interesting I didn't know about this.

https://nginx.org/en/docs/http/ngx_http_headers_module.html#expires

👍

Reply:
I didn't know about anything, so I googled ;)

`add_header Cache-Control "no-store, max-age=0";` also worked